### DISCUSSION TITLE
API Deprecated methods in Ampache 5

### DISCUSSION THREAD

**Discussion by manuelglez86** [ampache/ampache#2724] [General]
Hello.

Im back two days per week at work, I began to test the ampache 5 prerelease and I have noticed some changes on the API methods, tags method where deprecated and when I use the artist method the filter parameter doesnt work, where can I see the changes so I can adapt the API and continue testting?

Thanks

**Comment by manuelglez86** [comment]
I found the changes on the new wiki I will follow from here.

Thanks.

**Comment by usox** [comment]
Hi @manuelglez86 

Regarding your issue with the `artists` method - which method are you talking about exactly? `artists` or `artist`?

**Reply by manuelglez86** [reply]
Sorry, I use **artists** method but filter doesn´t work it shows all the artist on the database

**Reply by usox** [reply]
#2763

**Reply by usox** [reply]
And fixed.
